### PR TITLE
ApartApp - Submit application feature

### PR DESCRIPTION
**Pull request by NerdyGriffin**

### CODE REVIEW

File: src/views/ApartmentApp/components/StudentApplication/index.js [hunk 1]
```diff
@@ -535,15 +535,33 @@ const StudentApplication = ({ userProfile, authentication }) => {
...
+            // loadApplication(); //? Coming soon to a feature near you
+          }
+        })
+        .catch((err) => {
+          showGenericSubmitError();
+        });
     } else {
-      console.log('Not all applicants are valid'); //! DEBUG:
+      showGenericSubmitError();
     }
```
Comment (joshuaRogers24) [inline src/views/ApartmentApp/components/StudentApplication/index.js:565]: If this case occurs when the application ID has not yet been set, then should we not tell them to make sure they save their application first?

**Review by joshuaRogers24 — CHANGES_REQUESTED** [review]
Nice. Question, though...

File: src/views/ApartmentApp/components/StudentApplication/index.js (hunk 1 above)
Comment (NerdyGriffin) [reply src/views/ApartmentApp/components/StudentApplication/index.js:565]: The UI disables to button entirely in that case, so it should never get to that code without having an applicationID, unless something is actually broken

Comment (joshuaRogers24) [reply src/views/ApartmentApp/components/StudentApplication/index.js:565]: I see, this tests for the "actually broken" case.

**Review by joshuaRogers24 — APPROVED** [review]

**Pull request merged**